New EZ Office (Level 4); New LCZ Offices, Level 4 Command Sector Reorganization and Admin Panic Bunker

GGarciaECE opened the pull request:
## About the Pull Request
Create a new EZ Security section above the SD's office. LCZ Security offices are moved near the entry checkpoint of LCZ and reorganized. Level 4 Command section has been reorganized, a few offices were moved around and a new panic bunker has been created (more protective than the one on the surface). 

## Why It's Good For The Game
EZ Office was scrawny and awkward to use especially to process prisoners. Now they are closer to the SD's office and larger, with a proper detention center. 
LCZ office is closer to the checkpoint for ease of accessibility.
Panic Bunker allows command staff to flee to safety in the event of an SCP's breach. Field generator can keep SCPs away for so long as the bunker is powered (PACMANs generators with tritium fuel can help if the wire is cut). 

close #1121
close #1122

## Changelog

:cl:
add: Panic bunker on level 4 (under SD office) with multiple accesses within the floor.
add: EZ Security Office moved and expanded at Level 4 near the SD's office. 
add: LCZ security office moved and expanded near the LCZ entry checkpoint
add: ITD office and meeting room were moved around. ITD office is now near EZ Security office on level 4. 
/:cl:

GGarciaECE committed:
V1

GGarciaECE committed:
V1

GGarciaECE commented:
![image](https://user-images.githubusercontent.com/113508764/233680941-ec5f5c45-4546-4dfc-ab8e-21a7f694aeea.png)
![image](https://user-images.githubusercontent.com/113508764/233680954-f3ff6508-0035-4de4-848d-4fdb05825141.png)
![image](https://user-images.githubusercontent.com/113508764/233680966-7c388f83-80f9-4c47-b782-b3012f4db518.png)
![image](https://user-images.githubusercontent.com/113508764/233680979-e89e082c-f479-4d1f-a7f6-dbe1ab5c7c5b.png)

GGarciaECE committed:
V2 Fixing Map Check Errors

GGarciaECE committed:
V2 Fixing Map Check Errors

GGarciaECE committed:
V3

GGarciaECE committed:
V3

hry-gh reviewed (approved):
though i did always find escaping from admin dramatic, it always felt rather out of place

GGarciaECE committed:
V4 Misc modifs

GGarciaECE committed:
V4 Misc modifs

GGarciaECE committed:
V5 Fixed doors for tsuru

GGarciaECE committed:
V5 Fixed doors for tsuru

GGarciaECE committed:
Update (access hotfix and panic bunker tweaks)

GGarciaECE committed:
Update (access hotfix and panic bunker tweaks)

GGarciaECE committed:
Update site53-4.dmm

GGarciaECE committed:
Update site53-4.dmm

GGarciaECE commented:
Fixed some misc issues, added surgery kit to panic room, fixed door access for one LCZ office door, made panic room outter entry a bit more spacious etc

checkraisefold commented:
playing well on test merge

GGarciaECE committed:
Fix issues

GGarciaECE committed:
Fix issues

GGarciaECE committed:
Merge branch 'EZ/LCZ-office-mgiration,-Panic-Bunker,-ITD-Office-mgiration' of https://github.com/GGarciaECE/Foundation-19 into EZ/LCZ-office-mgiration,-Panic-Bunker,-ITD-Office-mgiration

Tsurupeta committed:
decals + lcz fixes + ez wall change

GGarciaECE committed:
Merge branch 'EZ/LCZ-office-mgiration,-Panic-Bunker,-ITD-Office-mgiration' of https://github.com/GGarciaECE/Foundation-19 into EZ/LCZ-office-mgiration,-Panic-Bunker,-ITD-Office-mgiration
